analytics_data_api/v0/urls/courses.py

```diff
@@ -16,6 +16,7 @@
     ('problems_and_tags', views.ProblemsAndTagsListView, 'problems_and_tags'),
     ('videos', views.VideosListView, 'videos'),
     ('reports/(?P<report_name>[a-zA-Z0-9_]+)', views.ReportDownloadView, 'reports'),
+    ('intervention', views.CourseInterventionView, 'intervention'),
```

Comment:
All of this data can be used by an instructor for the purpose of intervention. I would recommend naming this something that tells the API consumer more about what the data itself is.

Reply:
would 'intervention_engagements'  be better ?

Reply:
How about just "user_engagement"?